mne/bem.py

```diff
@@ -1346,11 +1340,21 @@ def write_bem_solution(fname, bem):
 # #############################################################################
 # Create 3-Layers BEM model from Flash MRI images
 
-def _prepare_env(subject, subjects_dir):
+def _prepare_env(subject, subjects_dir, requires_freesurfer, requires_mne):
     """Helper to prepare an env object for subprocess calls"""
     env = os.environ.copy()
+    if requires_freesurfer and not os.environ.get('FREESURFER_HOME'):
+        raise RuntimeError('FREESURFER_HOME environment variable not set. '
+                           'freesurfer is not setup.')
+    if requires_mne and not os.environ.get('MNE_ROOT'):
+        raise RuntimeError('MNE_ROOT environment variable not set. '
+                           'mne is not setup.')
```

Comment:
'I cannot find freesurfer, the FREESURFER_HOME environment variable not set.'

'I cannot find the MNE command line tools, the MNE_ROOT environment variable not set. '